launch/tier4_perception_launch/launch/object_recognition/detection/lidar_based_detection.launch.xml

```diff
@@ -59,16 +59,16 @@
   </group>
 
   <!-- CenterPoint -->
-  <group if="$(eval &quot;'$(var lidar_detection_model)'=='centerpoint'&quot;)">
+  <group if="$(eval &quot;'$(var lidar_detection_model)'=='centerpoint' or '$(var lidar_detection_model)'=='centerpoint_tiny'&quot;)">
     <push-ros-namespace namespace="centerpoint"/>
     <group>
       <include file="$(find-pkg-share lidar_centerpoint)/launch/lidar_centerpoint.launch.xml">
         <arg name="input/pointcloud" value="$(var input/pointcloud)"/>
         <arg name="output/objects" value="objects"/>
         <arg name="score_threshold" value="$(var score_threshold)"/>
-        <arg name="model_name" value="$(var centerpoint_model_name)"/>
-        <arg name="model_path" value="$(var centerpoint_model_path)"/>
-        <arg name="model_param_path" value="$(var centerpoint_model_param_path)"/>
+        <arg name="model_name" value="$(var lidar_detection_model)"/>
+        <arg name="model_path" value="$(find-pkg-share lidar_centerpoint)/data"/>
+        <arg name="model_param_path" value="$(find-pkg-share lidar_centerpoint)/config/$(var model_name).param.yaml"/>
```

Comment:
In this way, it is difficult to use customized model. 
I suppose these 3 lines should be able to change by args passed from from outside.

Reply:
IMO, these parameters belong to the Centerpoint. We are setting lidar detection models in "perception.launch.xml" and setting centerpoint-specific parameters in the same place. It looks a little bit confusing. 

 For customized models, we can directly change the model_name argument from "lidar_based_derection.launch.xml" or "camera_lidar_fusion_based_detection.launch.xml" doing the same job.

Reply:
>  these parameters belong to the Centerpoint.
IMO, It may be so, but may not.
Centerpoint model param might be different from the one available in autoware.universe if user fine-tune the weight in the project.
For this case, original structure allow user to pass other path directly. 

> For customized models, we can directly change the model_name argument from "lidar_based_derection.launch.xml" or "camera_lidar_fusion_based_detection.launch.xml" doing the same job.
I think, "lidar_based_derection.launch.xml" or "camera_lidar_fusion_based_detection.launch.xml" are not assumed to be changed in the project.

Reply:
> Centerpoint model param might be different from the one available in autoware.universe if user fine-tune the weight in the project.

Actually, I assumed, these 2 files could be used for fine-tuning/test purposes in the current PR; `"lidar_based_derection.launch.xml"` or `"camera_lidar_fusion_based_detection.launch.xml"`.

  I think the deployment arguments are more suitable to the "perception.launch.xml" file, instead to the test or fine-tuning arguments. It's a high-level perception control launch file. If there are situations where these parameters are not used, better to set them in place for users, especially unfamiliar people.

` 
 <arg name="centerpoint_model_name" default="centerpoint_tiny""/>
  <arg name="centerpoint_model_path" default="$(find-pkg-share lidar_centerpoint)/data"/>
  <arg name="centerpoint_model_param_path" default="$(find-pkg-share lidar_centerpoint)/config/$(var centerpoint_model_name).param.yaml"/>
`

Reply:
yes, of course. thanks for the explanation.

Reply:
I changed files as wanted